addons/website/static/src/scss/website.scss

```diff
@@ -214,6 +214,13 @@ $-seen-urls: ();
     }
 }
 
+$-header-text-color: o-website-value('header-text-color');
+#wrapwrap:not(.o_header_overlay), header.o_header_is_scrolled {
+    .nav-link > *, .nav-link::after {
```

Comment:
This selector targets all `nav-links` inside `#wrapwrap` and not just those in the navbar (e.g. snippet tabs is impacted)